nodes.py

```diff
@@ -203,6 +203,12 @@ def INPUT_TYPES(self):
                     "default": 1000,
                     "min": 1
                 }),
+            },
+            "optional": {
+                "EXAMPLE_IMAGE": ("IMAGE",),
```

Comment:
I use upper case names for inputs that are pure datatypes like "IMAGE", "STRING", etc.
I propose to rename these variables into:
```
"image_example": ("IMAGE",),
"response_example": ("STRING", {
```